src/server/wesnothd/player_connection.hpp

```diff
@@ -70,7 +70,7 @@ class player_record
 
 	void enter_lobby();
 
-	const std::chrono::time_point<std::chrono::steady_clock> login_time;
+	const std::chrono::system_clock::time_point login_time;
```

Comment:
Unsure about this change. It seems to make more sense as a system_time, but I also see the use for having it be monotonic. Though does it make sense to use a steady clock to format user-visible strings showing logged-in duration?

Reply:
After discussion on Discord, I'm going to revert this to steady_clock.